chord_metadata_service/restapi/api_renderers.py

```diff
@@ -117,18 +117,18 @@ def generate_csv_response(data, filename, columns):
     return response
 
 
-def render_individual_age(individual):
-    if "time_at_last_encounter" not in individual:
+def render_age(individual, time_key):
```

Comment:
add return type signature, and add type signature for `individual` (dict) and `time_key` (str)